libmpv/qml/main.qml

```diff
@@ -13,7 +13,7 @@ Item {
 
         MouseArea {
             anchors.fill: parent
-            onClicked: renderer.command(["loadfile", "test.mkv"])
+            onClicked: renderer.command(["loadfile", "C:/Users/Siddhant/Downloads/orig cut.mp4"])
```

Comment:
Please leave this as a generic `test.mkv` instead of a file on your own computer.